zone/zonedb.h

```diff
@@ -356,6 +356,8 @@ class ZoneDatabase : public SharedDatabase {
 	bool		DeleteCharacterCorpse(uint32 dbid);
 	bool		SummonAllCharacterCorpses(uint32 char_id, uint32 dest_zoneid, uint16 dest_instanceid, const glm::vec4& position);
 	bool		SummonAllGraveyardCorpses(uint32 cur_zoneid, uint32 dest_zoneid, uint16 dest_instanceid, const glm::vec4& position);
+	int			CountCharacterCorpses(uint32 char_id);
```

Comment:
For some reason we have some weird indents here

Reply:
Tried fixing it manually earlier, but couldn't do it with spaces or tabs.

Reply:
🤔